diff --git a/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_128x128x1x8.py b/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_128x128x1x8.py
index 1a377a5..fd6485e 100644
--- a/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_128x128x1x8.py
+++ b/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_128x128x1x8.py
@@ -28,7 +28,7 @@ import triton.language as tl
 
 try:
     # @manual=//triton:triton
-    from triton.language.extra.cuda.libdevice import fast_dividef
+    from triton.language.extra.libdevice import fast_dividef
 except ImportError:
     # pyre-ignore: Undefined import [21]
     # @manual=//triton:triton
diff --git a/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_32x32x2x2.py b/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_32x32x2x2.py
index 3faafc1..cfa7179 100644
--- a/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_32x32x2x2.py
+++ b/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_32x32x2x2.py
@@ -28,7 +28,7 @@ import triton.language as tl
 
 try:
     # @manual=//triton:triton
-    from triton.language.extra.cuda.libdevice import fast_dividef
+    from triton.language.extra.libdevice import fast_dividef
 except ImportError:
     # pyre-ignore: Undefined import [21]
     # @manual=//triton:triton
diff --git a/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_64x64x2x4.py b/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_64x64x2x4.py
index ee463ac..bf5a8cb 100644
--- a/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_64x64x2x4.py
+++ b/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_64x64x2x4.py
@@ -28,7 +28,7 @@ import triton.language as tl
 
 try:
     # @manual=//triton:triton
-    from triton.language.extra.cuda.libdevice import fast_dividef
+    from triton.language.extra.libdevice import fast_dividef
 except ImportError:
     # pyre-ignore: Undefined import [21]
     # @manual=//triton:triton
diff --git a/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_base.py b/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_base.py
index 3776131..0122dae 100644
--- a/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_base.py
+++ b/hammer/generative_recommenders/ops/triton/triton_ragged_hstu_attention_base.py
@@ -28,7 +28,7 @@ import triton.language as tl
 
 try:
     # @manual=//triton:triton
-    from triton.language.extra.cuda.libdevice import fast_dividef
+    from triton.language.extra.libdevice import fast_dividef
 except ImportError:
     # pyre-ignore: Undefined import [21]
     # @manual=//triton:triton
